docs/app/docs/data/item-group/page.mdx

```diff
@@ -0,0 +1,291 @@
+import {Callout} from "nextra/components";
```

Comment:
I think it generally looks good, but I would argue that we probably don't want users using `Item` and `ItemGroup` by itself. It could probably be better categorized as a foundation widget.

What are your thoughts? @Pante